libs/design-system-core/src/components/ErrorLoader/index.tsx

```diff
@@ -65,6 +71,9 @@ const ErrorLoader: React.FC<ErrorLoaderProps> = ({ children, ...props }) => {
     case 'list-not-available':
       imagesrc = `${publicImgPath}/list-not-available.webp`;
       break;
+    case 'empty-list':
```

Comment:
Ideally it would make sense to have one image to represent empty list. If not since we have 'list-not-available' we can rename it to `empty-list-1` and the one you added to `empty-list-2`. That way its easier to understand we have two variations of empty list. And even rename the files names.

Reply:
the idea is that list-not-available is an error (aka "Failed to fetch the list" and eventually have a retry button) whereas empty list simply means that the fetch succeded but there is nothing to show. Now that i think about it this is not actually an error so it does not belong to ErrorLoader... it's just very convenient to use it this way and not create yet a new component

Reply:
Yeah initially I created the `list-not-available` for errors only but for the same reasoning you described I also used it for queries which succeed but has empty data according to the design. I think we will need to revisit this, I went through ds core and saw we have `DefaultEmptyCard` which seems to be created for empty list.

Reply:
reverted the error-loader and replaced with `DefaultEmptyCard` component